jwst/superbias/superbias_step.py

```diff
@@ -8,31 +8,43 @@
 
 class SuperBiasStep(Step):
     """
-    SuperBiasStep: Performs super-bias subtraction by subtracting
-    super-bias reference data from the input science data model.
+    Perform super-bias subtraction.
+
+    Subtract super-bias reference data from the input science data model.
     """
 
     class_alias = "superbias"
 
     spec = """
-    """ # noqa: E501
+    """  # noqa: E501
 
-    reference_file_types = ['superbias']
+    reference_file_types = ["superbias"]
 
     def process(self, step_input):
-
+        """
+        Apply the superbias correction.
+
+        Parameters
+        ----------
+        step_input : data model object
```

Comment:
does this have to be a datamodel?  I think it could also be a string pointing to a filename containing a datamodel